### DISCUSSION TITLE
Is screen reader compatibility on Mac OS in the works

### DISCUSSION THREAD

**Discussion by deathdogg** [zed-industries/zed#8146] [Feature Requests]
hi,
I'm a completely blind dev, and I'm really curious about this project. I tend to use editors like visual studio code and Xcode, which provide amazing accessibility support. Are there any plans to make this IDE accessible on Mac OS using VoiceOver? I don't know what framework was used to render the application, but none of the elements are read out by the screen reader upon launching the application, so I'm not sure what can be done here. Thank you, and I'm looking forward to helping where I can to make this a reality

**Comment by Moshyfawn** [comment]
There's an issue #7895 tracking specifically VoiceOver support for Zed.

[GPUI](https://github.com/zed-industries/zed/tree/main/crates/gpui) is the framework that powers Zed.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Reply by sportfloh** [reply]
@JosephTLyons yet another tool to get familiar with! I'm really annoyed by the behaviour of your team. Closed issues with no reason, discussions/questions stay unanswered. Why should I eveny put more time into this project. At least a link to where this Disscussion on Discord would be nice! Please do your work more thoroughly in the future! Thx
